services/movie_session.py

```diff
@@ -0,0 +1,51 @@
+import init_django_orm  # noqa: F401
+import datetime
+
+from db.models import MovieSession, CinemaHall, Movie
+from django.db.models import QuerySet
+
+
+def create_movie_session(
+        movie_show_time: datetime,
+        movie_id: int,
+        cinema_hall_id: int
+) -> None:
+    MovieSession.objects.create(
+        show_time=movie_show_time,
+        cinema_hall=CinemaHall.objects.get(id=cinema_hall_id),
+        movie=Movie.objects.get(id=movie_id)
+    )
+
+
+def get_movies_sessions(session_date: str = None) -> QuerySet:
+    if session_date:
+        return MovieSession.objects.filter(show_time__date=session_date)
+    return MovieSession.objects.all()
+
+
+def get_movie_session_by_id(movie_session_id: int) -> QuerySet:
+    return MovieSession.objects.get(id=movie_session_id)
+
+
+def update_movie_session(
+        session_id: int,
+        show_time: datetime = None,
+        movie_id: int = None,
+        cinema_hall_id: int = None
+) -> None:
+    if show_time:
+        MovieSession.objects.filter(id=session_id).update(
+            show_time=show_time,
+        )
+    if movie_id:
+        MovieSession.objects.filter(id=session_id).update(
+            movie=movie_id,
+        )
+    if cinema_hall_id:
+        MovieSession.objects.filter(id=session_id).update(
+            cinema_hall=cinema_hall_id,
+        )

```

Comment:
No need to do 3 update statement do it only in 2 or 1 queries. Two possible options:
1. Get an object, set all fields, and save
2. Create some dict which field we need to update and then do update